src/components/bookList.js

```diff
@@ -1,39 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getAllBooks } from '../redux/books/books';
 import Book from './Book';
-
-const { v4: uuidv4 } = require('uuid');
+import InputBook from './inputBook';
 
 const BooksList = () => {
-  const books = [
-    {
-      id: uuidv4(),
-      title: 'Eloquent Javascript',
-      author: 'Marijn Haverbeke',
-    },
-    {
-      id: uuidv4(),
-      title: 'Lord of The Rings',
-      author: 'J. R. R. Tolkien',
-    },
-    {
-      id: uuidv4(),
-      title: 'Harry Potter',
-      author: 'J.K. Rowling',
-    },
-  ];
+  const dispatch = useDispatch();
+  const { booksReducer } = useSelector((state) => state);
+
+  useEffect(() => {
+    dispatch(getAllBooks());
+  }, [dispatch]);
 
   return (
 
-    <ul>
-      {books.map((book) => (
+    <div>
+      {booksReducer.map((book) => (
         <Book
-          id={book.id}
-          key={book.id}
-          title={book.title}
-          author={book.author}
+          key={book.item_id}
+          book={book}
         />
       ))}
```

Comment:
## [OPTIONAL]

- [ ] I would suggest you render a message when the is not book in the store

![image](https://user-images.githubusercontent.com/65068771/156897219-bc57ff0e-126b-4edf-ad1f-1c2df1a45b8d.png)

### Example: 

![image](https://user-images.githubusercontent.com/65068771/156897267-d48415e8-34cd-4b1f-915e-7215745362f7.png)